README.md

```diff
@@ -341,10 +341,13 @@ below for using each actor.
 
   * [aws.cloudformation.Create](docs/actors/aws.cloudformation.Create.md)
   * [aws.cloudformation.Delete](docs/actors/aws.cloudformation.Delete.md)
+  * [aws.elb.UseCert](docs/actors/aws.elb.UseCert.md)
```

Comment:
Sorry to do this so late -- but can we rename this to something like `UpdateCert` or `SetCert`? `UseCert` seems weird because you're not really actively using the cert, you're making a setting change to the ELB. Quick regex search through everything?